src/test/test_process_slow.c

```diff
@@ -324,7 +324,7 @@ test_callbacks_terminate(void *arg)
 }
 
 struct testcase_t slow_process_tests[] = {
-  { "callbacks", test_callbacks, TT_FORK, NULL, NULL },
-  { "callbacks_terminate", test_callbacks_terminate, TT_FORK, NULL, NULL },
+  { "callbacks", test_callbacks, 0, NULL, NULL },
```

Comment:
We should have  followup ticket to revert this change later on, once we can get our evloop module to handle forking better.

Reply:
Opened https://trac.torproject.org/projects/tor/ticket/28845